README.md

```diff
@@ -292,15 +292,18 @@ symbols include the following (as defined in `nvml.h`):
 
 ```
 #ifndef NVML_NO_UNVERSIONED_FUNC_DEFS
-    #define nvmlInit                            nvmlInit_v2
-    #define nvmlDeviceGetPciInfo                nvmlDeviceGetPciInfo_v3
-    #define nvmlDeviceGetCount                  nvmlDeviceGetCount_v2
-    #define nvmlDeviceGetHandleByIndex          nvmlDeviceGetHandleByIndex_v2
-    #define nvmlDeviceGetHandleByPciBusId       nvmlDeviceGetHandleByPciBusId_v2
-    #define nvmlDeviceGetNvLinkRemotePciInfo    nvmlDeviceGetNvLinkRemotePciInfo_v2
-    #define nvmlDeviceRemoveGpu                 nvmlDeviceRemoveGpu_v2
-    #define nvmlDeviceGetGridLicensableFeatures nvmlDeviceGetGridLicensableFeatures_v3
-    #define nvmlEventSetWait                    nvmlEventSetWait_v2
+    #define nvmlInit                                nvmlInit_v2
+    #define nvmlDeviceGetPciInfo                    nvmlDeviceGetPciInfo_v3
+    #define nvmlDeviceGetCount                      nvmlDeviceGetCount_v2
+    #define nvmlDeviceGetHandleByIndex              nvmlDeviceGetHandleByIndex_v2
+    #define nvmlDeviceGetHandleByPciBusId           nvmlDeviceGetHandleByPciBusId_v2
+    #define nvmlDeviceGetNvLinkRemotePciInfo        nvmlDeviceGetNvLinkRemotePciInfo_v2
+    #define nvmlDeviceRemoveGpu                     nvmlDeviceRemoveGpu_v2
+    #define nvmlDeviceGetGridLicensableFeatures     nvmlDeviceGetGridLicensableFeatures_v3
+    #define nvmlEventSetWait                        nvmlEventSetWait_v2
+    #define nvmlDeviceGetAttributes                 nvmlDeviceGetAttributes_v2
+    #define nvmlDeviceGetComputeRunningProcesses    nvmlDeviceGetComputeRunningProcesses_v2
+    #define nvmlDeviceGetGraphicsRunningProcesses   nvmlDeviceGetGraphicsRunningProcesses_v2
```

Comment:
I would technically expect this in the next commit, but it's OK here too.

Reply:
Updated.